client/src/components/atoms/dataIcon/dataIcon.jsx

```diff
@@ -0,0 +1,34 @@
+// Imports
+import * as Styles from './dataIcon.styles.js'
+
+// Component
+const dataIcon = ({ width, height, opacity, fill }) => {
```

Comment:
Functie is hier geschreven zonder hoofdletter, en wordt elders geimporteerd met hoofdletter